src/createJestConfig.ts

```diff
@@ -1,15 +1,30 @@
 import { Config } from '@jest/types';
+import { createRequire } from 'module';
+import * as Path from 'path';
```

Comment:
```suggestion
import path from 'path';
```

That is how it is imported in several files, so please be consistent.